README.md

```diff
@@ -6,7 +6,22 @@ If you're looking for Azure Service Management (ASM) please refer to this [repo]
 
 Additional info on Azure deployment models [https://docs.microsoft.com/azure/azure-resource-manager/resource-manager-deployment-model](https://docs.microsoft.com/azure/azure-resource-manager/resource-manager-deployment-model)
 
-## Azure Resource Management (preview)
+## Important Announcement
+
+As of February 2021, Azure Resource Management SDK for Ruby **has entered a retirement phase and is no longer officially supported by Microsoft**. Here is the complete list of packages that are affected by this. We are committed to making critical security and bug fixes for libraries in this repo until **12/31/2021**. After 2021, this repo will no longer be maintained.  
```

Comment:
"12/31/2021" -> "2021-12-31" or "December 31, 2021" because outside of the US "12/31" while possibly un-ambiguous would still look odd :)